manifests/action_flush_dispatcher_cache.pp

```diff
@@ -1,5 +1,5 @@
 class aem_curator::action_flush_dispatcher_cache (
-  $docroot_dir = $::docroot_dir,
+  $docroot_dir = lookup('common::docroot_dir'),
```

Comment:
Please move the lookup to the stack provisioner layer because the interface to this puppet module should be the parameters, it doesn't know anything about common.